server/pull_request.go

```diff
@@ -457,6 +468,44 @@ func (s *Server) CleanOutdatedPRs() {
 	mlog.Info("Finished update the outdated prs in the mattermod database....")
 }
 
+func (s *Server) CleanUpLabels(client *GithubClient, pr *model.PullRequest) {
+	labelsToRemove := []string{
```

Comment:
Could we move these to the config please?

Reply:
Done in e80b619cdbd873934459ac8284b547cb809c38ac